### ISSUE TITLE
HockeyApp Feedback Service

### ISSUE THREAD

**Issue by petero-dk** [Microsoft/AppCenter-SDK-DotNet#677]
In Hockey App there is the possibility to have a feedback service, we have started integration with AppCenter (not upgraded from HockeyApp) but would like to have user feedback in-app. 

It is not possible to have AppCenter and HockeyAppSDK sidebyside because of the crash reporting native libraries in iOS, however the AppCenter roadmap does not have any indication of the HockeyApp Feedback/Chat system.

**Comment by petero-dk** [comment]
I cannot see how to do that on Xamarin. The nuget package for HockeyApp contains all features and I see no way to disable individual packages.

**Comment by bahaa-ka** [comment]
we really need that feature to be also in app center
hockey app was a special solution because of this feature

**Comment by mathieumack** [comment]
I agree with @petero-dk , we already use feedback service in HockeyApp.
I can't migrate to AppCenter without this service !
Did you know when the HockeyApp platform will be decommissioned ?

**Comment by Costo** [comment]
For months we've been waiting for the announced feature-parity between with HockeyApp to migrate our apps to App Center. All our apps use the feedback feature from HockeyApp. We can't migrate to App Center and keep the Hockey App feedback feature since both SDKs can't run side-by-side. Are there any alternatives, other than "roll your own" feedback service ?

**Comment by ElektrojungeMS** [comment]
Thanks for the very valuable feedback on Feedback!

There might be a workaround for you. As of 5.1.2, there's a HockeyApp SDK Nuget that contains only Feedback functionality which would allow you to use App Center Analytics and Crashes and continue to use HockeyApp feedback.
With the next upcoming release of the HockeySDK-Xamarin, we are planning to release the Nuget through nuget.org, too.

**Comment by Costo** [comment]
@ElektrojungeMS Your link for "HockeyApp SDK Nuget" points to this issue.
Is there a new package other than https://www.nuget.org/packages/HockeySDK.Xamarin/ ?

**Comment by ElektrojungeMS** [comment]
Sorry about the wrong link. The package is not on nuget, it’s only available on our GH releases page: https://github.com/bitstadium/HockeySDK-Xamarin/releases/tag/5.1.2

**Comment by crazyfx1** [comment]
Since hockeyapp will be shut down in 1 year, are there any plans for the feedback service?

**Comment by elamalani** [comment]
@crazyfx1 We currently don't plan to support feedback service in App Center. We will update our [product roadmap](https://docs.microsoft.com/en-us/appcenter/general/roadmap) once we decide to support it. Thanks!

**Comment by projectgoav** [comment]
@crazyfx1 : They still don't support feature partiy with Xamarin Insights so you'll probably be waiting a while for HockeyApp parity....

**Comment by Loruki** [comment]
It seems that this functionality is still missing from the roadmap. Does anyone has a workaround ?

**Comment by elamalani** [comment]
Please follow this [document](https://docs.microsoft.com/en-us/appcenter/transition/feedback) to use other tools and replicate similar functionality in your app.
